tests/trade_reversions.test.ts

```diff
@@ -0,0 +1,185 @@
+import { expect } from 'chai'
+import { Mappings } from './wasm'
+import { ValueKind } from './wasm/runtime/ethereum'
+
+describe('onTradeReversion', () => {
+  const user = `0x${'1337'.repeat(10)}`
+  const orderId = 1337n
+  const orderUid = `${user}-${orderId}`
+  const tradeTxHash = `0x${'01'.repeat(32)}`
+  const reversionTxHash = `0x${'02'.repeat(32)}`
+  const logIndex = 1n
+  const tradeTimestamp = 10n * 300n + 42n
+  const reversionTimestamp = tradeTimestamp + 42n
+  const batchId = tradeTimestamp / 300n - 1n // Trade is recorded for the solved batch
+  const tradeId = `${tradeTxHash}-${logIndex}`
+  const sellToken = '0'
+  const sellVolume = 100000n * 10n ** 18n
+  const buyVolume = 500n * 10n ** 18n
+  const priceId = `${sellToken}-${batchId}`
+
+  async function createFixturesAndMakeTrade() {
```

Comment:
Slightly unrelated, but we could use this `createFixtures` pattern in `trades.spec.ts` as well, just so there is a consistent testing pattern.

Reply:
I'm not super happy with the unit testing as it is (lots of duplication across test files) and am thinking about creating a fixture module that we can share across our tests.

Reply:
Maybe we can create an issue so we don't forget to come back to this?

Reply:
https://github.com/gnosis/dex-subgraph/issues/125